applications/nrf_desktop/configuration/nrf52840_pca10059/mcuboot_ZDebugMCUBoot.conf

```diff
@@ -11,44 +11,17 @@ CONFIG_SYS_POWER_MANAGEMENT=n
 CONFIG_MAIN_STACK_SIZE=10240
 CONFIG_MBEDTLS_CFG_FILE="mcuboot-mbedtls-cfg.h"
 
+CONFIG_BOOT_ENCRYPT_RSA=n
```

Comment:
Do you want these changes in this PR?

Reply:
yes, I removed the size definitions in mcuboot config files and noticed that few lines were not needed or order was wrong.